pyiceberg/catalog/rest/__init__.py

```diff
@@ -130,6 +132,7 @@ class IdentifierKind(Enum):
 SNAPSHOT_LOADING_MODE = "snapshot-loading-mode"
 AUTH = "auth"
 CUSTOM = "custom"
+REST_SCAN_PLANNING_ENABLED = "rest-scan-planning-enabled"
```

Comment:
looks like `rest-scan-planning-enabled` is a catalog server config
https://github.com/apache/iceberg/blob/05998edb8808ae21739072cca17b621925a2081a/core/src/main/java/org/apache/iceberg/rest/RESTCatalogProperties.java#L40-L41